package.json

```diff
@@ -268,16 +268,20 @@
         "xml2js": "^0.6.2"
     },
     "devDependencies": {
+        "@eslint/eslintrc": "^3.3.1",
+        "@eslint/js": "^9.29.0",
         "@types/adm-zip": "^0.5.7",
         "@types/mocha": "^10.0.10",
         "@types/node": "^24.0.0",
         "@types/node-fetch": "^2.5.12",
         "@types/vscode": "^1.44.0",
         "@types/xml2js": "^0.4.8",
-        "@typescript-eslint/eslint-plugin": "^5.1.0",
-        "@typescript-eslint/parser": "^5.1.0",
+        "@typescript-eslint/eslint-plugin": "^8.35.0",
+        "@typescript-eslint/parser": "^8.35.0",
         "@vscode/test-electron": "^2.1.2",
-        "eslint": "^8.0.1",
+        "eslint": "^9.28.0",
+        "glob": "^11.0.3",
+        "globals": "^16.2.0",
```

Comment:
Just to understand: any specific reason why `@eslint/eslintrc`, `@eslint/js`, `glob`, and `globals` were added here?